src/mon/Monitor.cc

```diff
@@ -4741,7 +4757,9 @@ void Monitor::scrub_event_start()
     return;
   }
 
-  scrub_event = new C_Scrub(this);
+  scrub_event = new C_MonContext(this, [this](int) {
```

Comment:
might want to remove `C_Scrub` and `C_ScrubTimeout` in the header? i.e. we might need to backport #10513 also?